File: js/src/entity/ScheduleEntity.js

const { inspect } = require('node:util')

const { ThesmsworksEntityBase } = require('../ThesmsworksEntityBase')


// TODO: needs Entity superclass
class ScheduleEntity extends ThesmsworksEntityBase {

  constructor(client, entopts) {
    super(client, entopts)
    this.name = 'schedule'
    this.name_ = 'schedule'
    this.Name = 'Schedule'
  }


  make() {
    return new ScheduleEntity(this._client, this.entopts())
  }







}


module.exports = {
  ScheduleEntity
}

File: dart/lib/entity/ScheduleEntity.dart

// ignore_for_file: non_constant_identifier_names

import 'dart:async';

import '../ThesmsworksEntityBase.dart';
import '../utility/ErrUtility.dart';

// Typed models: see ../ThesmsworksTypes.dart (Schedule and the per-op request/match types).
class ScheduleEntity extends ThesmsworksEntityBase {
  ScheduleEntity(dynamic client, dynamic entopts) : super(client, entopts) {
    name = 'schedule';
    name_ = 'schedule';
    Name = 'Schedule';
  }

  ScheduleEntity make() {
    return ScheduleEntity(client, entopts());
  }





}
